src/workprover/Readme.md

```diff
@@ -0,0 +1,42 @@
+Please keep this module independent from the outside code, so that it can be reused in other applications.
```

Comment:
It's a Python package with the `__init__.py` file, and it's intended to be imported like `import workprover`. It could be moved to a separate repository, but I think, it's easier and safer to keep it in the same file tree.